setup.py

```diff
@@ -61,15 +61,28 @@ def run(self):
     # Build regular proto files.
     protos = glob.glob(os.path.join(self.indir, '*.proto'))
     if protos:
-      print "Attempting to build proto files:\n%s" % '\n'.join(protos)
-      subprocess.check_call([
+      print 'Attempting to build proto files:\n%s' % '\n'.join(protos)
+      cmd = [
           self.protoc,
           '--proto_path', self.indir,
           '--proto_path', self.protodir,
           '--python_out', self.outdir,
-          ] + protos)
+      ] + protos
+      try:
+        subprocess.check_call(cmd)
+      except OSError as e:
+        if e.errno == 2:
+          print 'You need to install the protobuf compiler.'
```

Comment:
'You may need to...' it's a bit of an assumption to make that ENOENT was definitely because of a missing protoc.